exercises/series/src/Example.hs

```diff
@@ -4,5 +4,6 @@ import Data.Char (digitToInt)
 import Data.List (tails)
 
 slices :: Int -> String -> [[Int]]
+slices 0 _ = [[]]
```

Comment:
The fact that I had to add the special case makes me wonder about it. As I noted earlier this solution would have said `slices 0 "012" = [[], [], []]` without it. Well, who am I to say that there should only be one subseries of size zero! Maybe it *should* be `[[], [], []]` instead. Maybe this is why it didn't get tested. Let's talk about it.